cs-commons/src/main/java/io/cloudslang/content/utils/BooleanUtilities.java

```diff
@@ -0,0 +1,61 @@
+package io.cloudslang.content.utils;
+
+import io.cloudslang.content.constants.BooleanValues;
+import io.cloudslang.content.constants.ExceptionsValues;
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/** A Boolean utility class that offers boolean string validation and conversion
+ * Created by victor on 31.08.2016.
+ */
+public final class BooleanUtilities {
```

Comment:
I want to be consistent with the other classes, to not have *Utils and *Utilities in the same package